modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -248,6 +348,29 @@ The operation generates a foreground mask.
 */
 GAPI_EXPORTS GMat BackgroundSubtractor(const GMat& src, const cv::gapi::video::BackgroundSubtractorParams& bsParams);
 
+/** @brief Standard Kalman filter algorithm. The operation uses standard matrices
+by default <http://en.wikipedia.org/wiki/Kalman_filter>.
+transitionMatrix, controlMatrix and measurementMatrix can be modified to get an
+extended Kalman filter functionality.
+
+@return Output image is predicted or corrected state, i.e. 32-bit or 64-bit float
+matrix @ref CV_32F or CV_64F.
```

Comment:
Please, add `@ref` before `CV_64F` to make a doxygen reference. See: [the documentation built](https://pullrequest.opencv.org/buildbot/export/pr/18869/docs/d4/d55/group__gapi__video.html#ga2e345448853cf133e5bc9e4b3a914f91)

Reply:
Done.